### PR TITLE
[RFR] Copostgres 0.1.0

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: src/api/order-products/orderProductModel.js [hunk 1]
```diff
@@ -1,42 +1,21 @@
...
-            FROM ${tableName}
-            WHERE order_id = $orderId`;
+    return Object.assign({}, orderProductModelClient, {
+        selectByOrderId,
+    });
+}
 
-        return await client.query({ sql, parameters: { orderId } });
-    };
+orderProductModel.queries = orderProductQueries;
```
Comment (Kmaschta) [inline src/api/order-products/orderProductModel.js]: What's the point of this line? Attach queries to a function?

Comment (djhi) [inline src/api/order-products/orderProductModel.js]: Aren't they already available directly on `orderProductModel` ?

Comment (ThieryMichel) [inline src/api/order-products/orderProductModel.js]: It is to expose the configurable queries.

Comment (djhi) [inline src/api/order-products/orderProductModel.js]: Those available directly on the model are already configured ?

**Pull request merged**